src/test/obj_recovery/obj_recovery.c

```diff
@@ -115,6 +121,23 @@ main(int argc, char *argv[])
 				TX_ADD_FIELD(D_RW(root)->foo, bar);
 
 				D_RW(D_RW(root)->foo)->bar = BAR_VALUE * 2;
+
+				/*
+				 * Even though flushes are not required inside
+				 * of a transaction, this is done here to
+				 * suppress irrelevant pmemcheck issues, because
+				 * we exit the program before the data is
+				 * flushed, while preserving any real ones.
+				 */
+				pmemobj_persist(pop,
```

Comment:
I'd rather not, it's important that we flush all the relevant things while recovering ;)